snapcraft/storeapi/_upload.py

```diff
@@ -33,6 +38,7 @@
 )
 
 
+logging.getLogger("requests").setLevel(logging.WARNING)
```

Comment:
we generally call the logger __file__. Any special reason for using "requests" here instead?